config/dynamic.js

```diff
@@ -41,7 +41,7 @@ export const app = {
 		},
 		enable: true,
 		checkFakeAuth,
-		apiAudience: process.env.ALTERNATE_API_AUDIENCE || `https://api.${baseUrl}/graphql`,
+		apiAudience: process.env.ALTERNATE_API_AUDIENCE || `https://gateway.${baseUrl}/graphql`,
```

Comment:
Noting that the changed value was a default for prod, but we're already overriding with the process.env in the prod environment config so this should be ok.

Reply:
Most of the other default values in this file are for development instead of production.